mentat/sampler/README.md

```diff
@@ -0,0 +1,23 @@
+# Mentat Sampler API
+The Sampler API is an open-source standard to captures interactions between a developer and an LLM-based AI Assistant. It's intended to facilitate sharing of benchmarks and fine-tuning data throughout the open-source community and industry.
+
+| Field            | Type          | Description |
+|------------------|---------------|-------------|
+| title            | `str`         | plaintext by creator |
+| description      | `str`         | plaintext by creator |
+| id               | `uuid`        |  |
```

Comment:
Why do we need an `id` at all? Isn't a title enough? I guess we plan to make so many (eventually) that we won't have time to name them?

Reply:
I wouldn't want to rely on names being unique - esp if it's public.